config/syntax/c

```diff
@@ -255,12 +255,13 @@ state error-ident error
 
 list keyword \
     if else for while do continue switch case break default return goto \
-    asm const static inline register extern volatile auto restrict sizeof \
-    typedef
+    asm const static inline register export extern volatile auto restrict \
+    sizeof typedef
 
 list c11-keyword \
-    _Alignas _Alignof _Noreturn \
-    alignas alignof noreturn
+    _Alignas _Alignof _Bool _Complex _Generic _Imaginary \
```

Comment:
`_Bool`, `_Complex` and `_Imaginary` are already in `list type` below.

I left `static_assert` out of this list deliberately because I personally find highlighting it like a keyword to be visually distracting. I don't mind adding (optional) support for highlighting it, but the reason should be that someone actually wants this behaviour, rather than just for the sake of completing the list of keywords.

Reply:
What about `static_assert` in `cpp-keyword` list?

Reply:
`static_assert` is part of C too, so I don't think it should be part of `cpp-keyword`. The best way to make it optional would be to add it to its own separate list, e.g.

```
list static-assert _Static_assert static_assert
```

...and integrate it into the main `inlist` rules.

I'd suggest trying it out in a branch for a while though. In a codebase with lots of static assertions, I just found it really annoying when I tried it.